src/wallet/test/coinselector_tests.cpp

```diff
@@ -1182,7 +1182,14 @@ BOOST_AUTO_TEST_CASE(coin_grinder_tests)
             }
             return available_coins;
         });
-        BOOST_CHECK(res);
+        SelectionResult expected_result(CAmount(0), SelectionAlgorithm::CG);
+        for (int i = 0; i < 10; i++) {
+            add_coin(2 * COIN, i,expected_result);
+        }
+        for (int j = 0; j < 17; ++j) {
+            add_coin(0.33 * COIN, j + 10, expected_result);
```

Comment:
nit: there seems to be a whitespace needed before expected result in the first loop. please run a formatter over the code if not done already.

Reply:
> nit: there seems to be a whitespace needed before expected_result in the first loop.

Thanks, done.

> please run a formatter over the code if not done already.

Do you mean `git diff -U0 HEAD~1.. | ./contrib/devtools/clang-format-diff.py -p1 -i -v` ?

I get an error after running, maybe my version fo clang-format is different.  

```
Formatting src/wallet/test/coinselector_tests.cpp
/home/yancy/Git/bitcoin/src/.clang-format:46:33: error: invalid boolean
BreakBeforeConceptDeclarations: Always
                                ^~~~~~
Error clang-format: Invalid argument
```

I'm using `Debian clang-format version 14.0.6`.  Maybe I'll open an issue for this.